src/legacy/core_plugins/timelion/public/services/saved_sheets.ts

```diff
@@ -56,5 +49,13 @@ savedSheetLoader.loaderProperties = {
   nouns: 'saved sheets',
 };
 
+// Register this service with the saved object registry so it can be
+// edited by the object editor.
+savedObjectManagementRegistry.register({
```

Comment:
Not sure why we register this, as timelion sheets don't show up (on master as well) in the Saved Objects management.
Is this needed at all?

Reply:
I've tested it, it's not needed anymore, I've removed it, thx